iib/workers/tasks/build_regenerate_bundle.py

```diff
@@ -457,7 +486,7 @@ def _replace_csv_pullspecs(bundle_metadata, replacement_pullspecs):
     for old_pullspec, new_pullspec in replacement_pullspecs.items():
         log.debug(
             '%s will be replaced with %s in the bundle CSVs',
-            old_pullspec.to_str(),
+            old_pullspec,
```

Comment:
Why was this changed?

Reply:
Type of the `old_pullspec` variable is already str.